### PR TITLE
Update autentication and watermark formating

### PR DESCRIPTION
**Pull request by mickotomic**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -7,3 +7,5 @@ BASE_URL=http://localhost:3000
 BASE_THUMBNAIL_SIZE=285x190
 JWT_SECRET=
 JWT_EXPIRATION_TIME=
+IMAGES_PATH=
```
Comment (stefanj122) [inline .env.dist]: You do't need this IMAGE_PATH and WATERMARK_PATH.

File: src/authentication/dto/registerUser.dto.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { IsString, Matches } from 'class-validator';
-import { IsBoolean, IsEmail } from 'class-validator';
+import { IsEmail } from 'class-validator';
```
Comment (stefanj122) [inline src/authentication/dto/registerUser.dto.ts]: This can be imported in one line.

**Review by Branimir — APPROVED** [review]
I could not find any major issues with this PR, please merge :). We might refactor some stuff later

File: src/types/payloadType.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-    displayName: string,
-    role: string,
-}
\ No newline at end of file
+  id: number;
+  firstName: string;
+  lastName: string;
+  displayName: string;
+  role: string;
+};
```
Comment (miroslavrs81) [inline src/types/payloadType.ts:7]: Members of type ends with , not with ; 
and last member role: string doesnt need to be ended with ,
export type JwtPayloadType = {
    id: number,
    firstName: string,
    lastName: string,
    displayName: string,
    role: string
};

Comment (mickotomic) [reply src/types/payloadType.ts:7]: Maybe... ;)

**Pull request merged**